_data/hosting.yml

```diff
@@ -71,6 +71,13 @@ websites:
       twitter: iPage
       img: ipage.png
       tfa: No
+      
+    - name: Mammoth Networks
+      url: https://www.mammoth.net.au/
+      img: mammothnetworks.png
+      tfa: Yes
+      software: yes
```

Comment:
Fixed the 'Yes' thing (sorry!)

Re: the doc link, there's nothing with any more information that's obvious.
If there's anything specific that you like seeing on Doc links I should be
able to get it added!

-- 
dave

On Tue, Aug 18, 2015 at 7:54 PM, Carlgo11 <notifications@github.com> wrote:

> In _data/hosting.yml
> <https://github.com/jdavis/twofactorauth/pull/1285#discussion_r37367567>:
>
> > @@ -71,6 +71,13 @@ websites:
> >        twitter: iPage
> >        img: ipage.png
> >        tfa: No
> > +
> > +    - name: Mammoth Networks
> > +      url: https://www.mammoth.net.au/
> > +      img: mammothnetworks.png
> > +      tfa: Yes
> > +      software: yes
>
> Change to Yes please.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jdavis/twofactorauth/pull/1285/files#r37367567>.
>